### ISSUE TITLE
v3.0 gets higher CER and WER than v2.1 while training on custom dataset

### ISSUE THREAD

**Issue by fatemeshiravand** [SeanNaren/deepspeech.pytorch#652]
Hi @SeanNaren 

I've been using v2.1 for a long time to train the model on my own dataset. This version has some instability issue and gets NAN loss soon after few epochs of training.
V3.0 seems to be more stable than v2.1, but while training on my own dataset, it gets very much higher CER and WER than v2.1 that makes this version (v3.0) unusable in my case.
I wanted to ask if this is normal or if I'm doing something wrong about the process of training v3.0 model.

Thanks for your attention. I’m looking forward to your reply.

**Comment by DatNgoBK** [comment]
I meet the same problem

**Comment by kouohhashi** [comment]
Have some one compared nn.CTCLoss and warp-ctc?
I have been using training loop based on v2 and I switched to nn.CTCLoss because warp-ctc stop compiling on my environment. 
But after switching to nn.CTCLoss my loss does not decrease as I expected.
I just think it might be because of loss function... and It seems warp-ctc still is better than nn.CTCLoss...

**Comment by navidnadery** [comment]
I got totally different results. In my case, the training loss/error

**Comment by navidnadery** [comment]
I got totally different results. In my case, v3 is much better on test data, while v2.1 is only good on train and validation data.
I think it totally depends on the data.
